### PR TITLE
Change FULL for ORIGINAL [4.7]

### PR DESCRIPTION
**Pull request by ildyria**
Associated with https://github.com/LycheeOrg/Lychee/pull/1618

### CODE REVIEW

**Comment by ildyria** [comment]
This one will make @nagmat84 happy: switching from FULL to ORIGINAL :)

**Review by qwerty287 — APPROVED** [review]
Untested

**Comment by ildyria** [comment]
> Untested

To test it you will need to use the php8.1 branch :)

File: scripts/main/photo.js [hunk 1]
```diff
@@ -1076,7 +1076,7 @@ photo.getArchive = function (photoIDs, kind = null) {
 	const myPhoto = photo.json && photo.json.id === photoIDs[0] ? photo.json : album.getByID(photoIDs[0]);
 
 	const kind2VariantAndLocalizedLabel = {
-		FULL: ["original", lychee.locale["PHOTO_FULL"]],
+		ORIGINAL: ["original", lychee.locale["PHOTO_FULL"]],
```
[the remark this answers is not in the document]
Comment (ildyria) [reply scripts/main/photo.js]: Done: [74efa27](https://github.com/LycheeOrg/Lychee-front/pull/361/commits/74efa27eb2676531c51d4b303a4336d4d36a955a)

**Review by qwerty287 — APPROVED** [review]

**Pull request merged**